src/main/java/core/basesyntax/impl/StorageImpl.java

```diff
@@ -3,17 +3,37 @@
 import core.basesyntax.Storage;
 
 public class StorageImpl<K, V> implements Storage<K, V> {
+    private static final int MAX_SIZE = 10;
+    private final Object[] keys = new Object[MAX_SIZE];
+    private final Object[] values = new Object[MAX_SIZE];
+    private int size = 0;
+
     @Override
     public void put(K key, V value) {
+        for (int i = 0; i < size; i++) {
+            if ((keys[i] == key) || (keys[i] != null && keys[i].equals(key))) {
```

Comment:
```suggestion
            if (keys[i] == key || keys[i] != null && keys[i].equals(key)) {
```
Also, actually, this comparison is duplicated within another method. Could you please move it to a separate method and call it whenever needed?